source-build.slnf

```diff
@@ -12,11 +16,9 @@
       "src\\Cli\\Microsoft.DotNet.Cli.Utils\\Microsoft.DotNet.Cli.Utils.csproj",
       "src\\Cli\\Microsoft.DotNet.Configurer\\Microsoft.DotNet.Configurer.csproj",
       "src\\Cli\\Microsoft.DotNet.InternalAbstractions\\Microsoft.DotNet.InternalAbstractions.csproj",
+      "src\\Cli\\Microsoft.TemplateEngine.Cli\\Microsoft.TemplateEngine.Cli.csproj",
+      "src\\Cli\\dotnet-new3\\dotnet-new3.csproj",
```

Comment:
This is not needed because this it's just used by a test DotnetNew3CompleteTests.
@vlada-shubina please help to double confirm.

Reply:
Correct. `dotnet-new3` is a test project and should not be shipped.

Reply:
Thanks for the feedback and pushing a fix.